packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -372,7 +372,7 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
 }
 
 // @public
-export interface CallCompositeStrings {
+export interface CallCompositeStrings extends CommonCompositeStrings {
```

Comment:
Ordinarily this is exactly what we should do! But the AzureReviewBoard have asked us to keep interfaces as flat as possible so for intellisense and when a developer looks at the type they get the whole interface and not have to find the extended interface and look at that as well

Reply:
Ah interesting! That's good to know, thanks James! 
It would have definitely made common composite files easier to work with if we had a common set of strings to access, that is unfortunate.